lang/java/tools/pom.xml

```diff
@@ -145,7 +145,7 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hadoop</groupId>
-      <artifactId>hadoop-core</artifactId>
```

Comment:
Adding hadoop-core doesnt seems to be working, it tries to find the jar which is not available. And for the given hadoop version (2.6.0) hadoop-client works fine.